rs/sns/testing/README.md

```diff
@@ -44,14 +43,183 @@ To run the scenario on the local PocketIC instance:
+   ```
+
+Once these steps are completed, the PocketIC will expose the IC network HTTP endpoint on "http://127.0.0.1:8080".
+`sns-testing` identity will be added as a hotkey to the NNS neuron with the majority voting power, so all NNS proposals made
+by `sns-testing` will be automatically accepted.
+
+Additionally, `nns-init` will output the ID of the NNS neuron that should be used to create NNS proposals.
+```
+...
+Use the following Neuron ID for further testing: 1
+```
+
+### Create the new SNS
+
+Now you can create a new SNS via NNS proposal.
+
+The suggested way to do this is to use `dfx sns propose` command. For more information please refer to the [documentation](https://internetcomputer.org/docs/building-apps/governing-apps/launching/launch-steps-1proposal#3-submit-nns-proposal-to-create-sns).
+
+Make sure to use `sns-testing` identity when creating the proposal.
+
+<details>
+<summary>Sample SNS proposal creation workflow</summary>
+<br>
+
+The example will use `//rs/sns/testing:sns_testing_canister` canister as SNS-controlled canister and will base on the [init YAML file from SNS CLI](../cli/test_sns_init_v2.yaml).
+
+**Make sure to use `start_time: null` in the swap parameters to ensure that the swap starts right away after the NNS proposal is executed.**
+
+1) Build and deploy `test` canister:
+   ```
+   bazel build //rs/sns/testing:sns_testing_canister
+   # 'r7inp-6aaaa-aaaaa-aaabq-cai' is the NNS Root canister
+   dfx canister --network http://127.0.0.1:8080 --identity sns-testing create test --controller r7inp-6aaaa-aaaaa-aaabq-cai --controller sns-testing --no-wallet
+   dfx canister --network http://127.0.0.1:8080 --identity sns-testing install test --wasm "$(bazel info bazel-bin)/rs/sns/testing/sns_testing_canister.wasm.gz"
+   ```
+
+2) Adjust init YAML file (you will need [`yq`](https://github.com/mikefarah/yq) to be installed to do this):
+   ```
+   yq -i ".dapp_canisters |= [\""$(dfx canister --network http://127.0.0.1:8080 id test)"\"]" ../cli/test_sns_init_v2.yaml
+   yq -i ".Distribution.Neurons[0].principal |= \""$(dfx identity get-principal --identity sns-testing)"\"" ../cli/test_sns_init_v2.yaml
+   yq -i ".Swap.start_time |= null" ../cli/test_sns_init_v2.yaml
+   ```
+3) Propose to create the new SNS:
+   ```
+   pushd ../cli
+   # //rs/sns/cli:sns doesn't support CLI-provided identities despite '--identity' option
+   dfx identity use sns-testing
+   bazel run //rs/sns/cli:sns -- propose --network http://127.0.0.1:8080 --neuron-id 1 $PWD/test_sns_init_v2.yaml
+   popd
+   ```
+
+4) Complete the swap for the newly created SNS
+   ```
+   SNS_NAME="$(yq -r .name ../cli/test_sns_init_v2.yaml)"
+   bazel run //rs/sns/testing:sns-testing -- --network http://127.0.0.1:8080 swap-complete --sns-name "$SNS_NAME"
+   ```
+</details>
+
+Once the NNS proposal to create the new SNS is adopted and executed, the SNS swap is supposed to open.
```

Comment:
This should probably go between steps 3 and 4

Reply:
The steps above are hidden in "Sample SNS proposal creation workflow" dropdown. This dropdown showcases an example of SNS creation based on `test_sns_init_v2.yaml`.

While this line introduces the `swap-complete` subcommand interface, which should be agnostic to the specific SNS unlike the instructions above